snap/snapcraft.yaml

```diff
@@ -90,6 +90,15 @@ parts:
     stage:
     - -linux
     - -src
+    override-build: |
+      snapcraftctl build
+      if [ -d "../install/linux/lib64" ]; then
+        cd ../install/linux/lib64
+      else 
+        cd ../install/linux/lib32
+      fi
```

Comment:
There should probably be 2 completely independent `if` checks for `lib32` and `lib64` (i.e. don't assume it's one or the other, assume that it could be either or both).  That way, if we update the package to include both 32-bit and 64-bit libs, it will continue to work without needing any modification.

Reply:
Also, probably best to avoid `cd` and instead just use the path when creating the symlink (if that makes sense).

Finally: should this really be an `override-build` or would it be better as an `override-install`?  That might avoid the need to peek into the higher-level dir with `..`.

```
override-install: |
  snapcraftctl install
  # create the symlink
```

Reply:
I don't think there is an override-install

Reply:
> I don't think there is an override-install

Ah, no worries.  There is an `install`, but I can't remember if that's deprecated or not.  In any case, can you update to have two separate `if` checks, so that we don't have to worry about whether the package gets updated to contain both `lib64` and `lib32` ... ?

Reply:
unfortunately, install has been deprecated

Reply:
I modified the scriptlet, now I think it looks good